packages/sdk/src/actor/behaviors/behaviorTypes.ts

```diff
@@ -11,4 +11,6 @@ export type BehaviorType
 	= 'none'
 	| 'target'
 	| 'button'
+	| 'tool'
```

Comment:
`BehaviorType` is public, no? Don't add 'tool' or 'pen' values until we're ready to make them available.